types/src/shared/utils/string_utils.ts

```diff
@@ -69,3 +69,7 @@ export function redactString(str: string, n: number) {
   const redacted = "•".repeat(str.length - n) + str.slice(-n);
   return redacted;
 }
+
+export function truncate(text: string, length: number, omission = "...") {
```

Comment:
We must already have this somewhere, should try to dry it? I think we have one in connectors, now that it's here it should be shared?

Reply:
It does not exist, everyone went with there own `.substring()` implementation 🙃.